openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -281,6 +282,10 @@ def get_plugin_info(self):
 
         plugin_payload = attr.asdict(plugin_info)
 
+        if instance.data["renderer"] == "arnold":
+            plugin_payload["ArnoldVerbose"] = cmds.getAttr(
+                "defaultArnoldRenderOptions.log_verbosity")
```

Comment:
i have tried to include this in `_get_arnold_render_payload `but it seems it doesn't collect the attributes when I checked with the plugin info. And also if I use `get_attr_in_layers`, it won't return the enum option, of which I get the string value of `debug`(which is also right, but cmds.getattr would be returning integer) . I am still thinking what the best way is to use `get_attr_in_layers` while I can validate right.

Reply:
You could add `as_string=True` as an argument to `get_attr_in_layer` 

And then change all the `asString=True` usage in the function to `asString=as_string`.
Then you can call the function as `get_attr_in_layer(attr, layer, as_string=False)` and voila?

Reply:
> i have tried to include this in _get_arnold_render_payload but it seems it doesn't collect the attributes when I checked with the plugin info

Please try to figure out why that is the case - because that sounds like a bug of its own that we should solve instead.

Reply:
> Ah wait, `_get_arnold_render_payload` likely is ONLY used for exporting ASS files, not for rendering?

yes it is as far as I know and the verbosity level check is legit for both ass and general image rendering in Arnold.